src/pages/homePage/HomePage.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { useSession } from 'hooks';
+import { useMediaQuery } from 'react-responsive';
+
+import LogoutButton from 'components/user/LogoutButton';
+import MainLayout from 'components/common/MainLayout';
+import Map from 'components/common/Map/Map';
+import './homePage.scss';
+
+const HomePage = () => {
+  const { user } = useSession();
+
+  const isTabletOrMobile = useMediaQuery({ query: ' (max-width: 960px)' });
+  const isPortrait = useMediaQuery({ query: '(orientation: portrait)' });
+
+  return (
+    <>
+      {user &&
+        user.email &&
```

Comment:
```suggestion
      {user?.email &&
```

Here you can take advantage of the [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)